compute/cloud-client/src/main/java/compute/snapshotschedule/CreateSnapshotSchedule.java

```diff
@@ -62,66 +64,42 @@ public static Status createSnapshotSchedule(String projectId, String region,
             String snapshotScheduleName, String scheduleDescription, int maxRetentionDays,
             String storageLocation, String onSourceDiskDelete)
           throws IOException, ExecutionException, InterruptedException, TimeoutException {
-    String startTime = "08:00";
-    ResourcePolicySnapshotSchedulePolicySnapshotProperties snapshotProperties =
-            ResourcePolicySnapshotSchedulePolicySnapshotProperties.newBuilder()
-                    .addStorageLocations(storageLocation)
-                    .build();
-
-    // Define the hourly schedule:
-    int snapshotInterval = 10; // Create a snapshot every 10 hours
-    ResourcePolicyHourlyCycle hourlyCycle = ResourcePolicyHourlyCycle.newBuilder()
-            .setHoursInCycle(snapshotInterval)
-            .setStartTime(startTime)
-            .build();
-
-    // Define the daily schedule.
-    // ResourcePolicyDailyCycle dailySchedule =
-    //         ResourcePolicyDailyCycle.newBuilder()
-    //                 .setDaysInCycle(1)  // Every day
-    //                 .setStartTime(startTime)
-    //                 .build();
-
-    // Define the weekly schedule.
-    // List<ResourcePolicyWeeklyCycleDayOfWeek> dayOfWeeks = new ArrayList<>();
-    // ResourcePolicyWeeklyCycleDayOfWeek tuesdaySchedule =
-    //         ResourcePolicyWeeklyCycleDayOfWeek.newBuilder()
-    //                 .setDay(ResourcePolicyWeeklyCycleDayOfWeek.Day.TUESDAY.toString())
-    //                 .setStartTime(startTime)
-    //                 .build();
-    // dayOfWeeks.add(tuesdaySchedule);
-    //
-    // ResourcePolicyWeeklyCycle weeklyCycle = ResourcePolicyWeeklyCycle.newBuilder()
-    //         .addAllDayOfWeeks(dayOfWeeks)
-    //         .build();
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (ResourcePoliciesClient resourcePoliciesClient = ResourcePoliciesClient.create()) {
+      String startTime = "08:00";
+      // Define the hourly schedule:
+      int snapshotInterval = 10; // Create a snapshot every 10 hours
```

Comment:
Why are there two different comment styles? Kindly move the comment on line 72 to top.

Reply:
Fixed styles.